main.go

```diff
@@ -132,6 +134,17 @@ func stalePullRequests(staleTime time.Duration) (stale []Issue, err error) {
 	return stale, nil
 }
 
+type VacationUsers []string
```

Comment:
You could have used a `map[string]bool` for this and avoided the `Contains` method.